worker/build.rs

```diff
@@ -78,25 +82,36 @@ fn main() {
     #[cfg(target_os = "windows")]
     {
         if !std::path::Path::new("worker/out/msys/bin/make.exe").exists() {
-            let python = if Command::new("where")
-                .arg("python3.exe")
+            let python = if let Ok(python) = env::var("PYTHON") {
+                python
+            } else if Command::new("where")
+                .arg("python3")
                 .status()
                 .expect("Failed to start")
                 .success()
             {
-                "python3"
+                "python3".to_string()
             } else {
-                "python"
+                "python".to_string()
             };
 
+            let dir = format!("{}/msys", mediasoup_out_dir.replace('\\', "/"));
+
             if !Command::new(python)
                 .arg("scripts\\getmake.py")
+                .arg("--dir")
+                .arg(dir.clone())
                 .status()
                 .expect("Failed to start")
                 .success()
             {
                 panic!("Failed to install MSYS/make")
             }
+
+            env::set_var(
+                "PATH",
+                format!("{}\\bin;{}", dir, env::var("PATH").unwrap()),
```

Comment:
Is it OK to use back slash here?

Reply:
Yes, it was already the same some lines below where `env::set_var()` is also called.